scripts/regression_test_python.py

```diff
@@ -348,10 +389,11 @@ def test_call_and_select_statements():
 
 
 def main():
-    test_tpch()
-    test_arrow_dictionaries_scan()
-    test_loading_pandas_df_many_times()
-    test_call_and_select_statements()
+    # test_tpch()
+    # test_arrow_dictionaries_scan()
+    # test_loading_pandas_df_many_times()
```

Comment:
I have made this mistake as well, please uncomment the other benchmarks again 😉

Reply:
woopsie haha, thanks!